src/js/home/LayersPanel.jsx

```diff
@@ -23,11 +23,14 @@ const LayerCheckbox = styled.input`
   accent-color: ${({ layerColor }) => layerColor || "var(--teal-1)"};
 `;
 
-const sliderThumb = (layerColor) => css`
-  background: var(--white);
+const sliderThumb = (layerColor, sliderShape, isSliderColorInverted) => css`
   border: 4px solid;
```

Comment:
Let's change this back to a `2px` border so that it matches Andrea's style guide.
```suggestion
  border: 2px solid;
```
Here's what that looks like:
![Screenshot from 2022-09-28 08-05-33](https://user-images.githubusercontent.com/40574170/192815515-9bd9b8cc-ce3d-4e67-b3d8-c6328d40ab90.png)

Reply:
It is not look and hard to distinguish when the colors are inverted (like now) and we are not following the sizes pixel by pixel in any other stuff plus we are using different shapes and all.